<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(211, 215, 207);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>1046</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>SARP Messages</string>
     </attribute>
     <widget class="QCheckBox" name="messages_auto_scroll_box">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>320</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Auto scroll</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="sarp_message_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1021</width>
        <height>306</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>601</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Filter message options</string>
      </property>
      <widget class="QCheckBox" name="filter_realtime_box">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>25</y>
         <width>76</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Realtime</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="filter_playback_box">
       <property name="geometry">
        <rect>
         <x>95</x>
         <y>25</y>
         <width>76</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Playback</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="filter_playback_box_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>25</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Long</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="filter_playback_box_3">
       <property name="geometry">
        <rect>
         <x>245</x>
         <y>25</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Short</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>496</width>
      <height>271</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>SARP RAW Frames</string>
     </attribute>
     <widget class="QCheckBox" name="frames_auto_scroll_box">
      <property name="geometry">
       <rect>
        <x>385</x>
        <y>205</y>
        <width>86</width>
        <height>23</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Auto scroll</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="sarp_frame_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>466</width>
        <height>196</height>
       </rect>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Deframer controls</string>
    </property>
    <widget class="QComboBox" name="message_format_box">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>34</y>
       <width>126</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>35</y>
       <width>46</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Mode:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>105</y>
      <width>246</width>
      <height>166</height>
     </rect>
    </property>
    <property name="title">
     <string>Decoder status</string>
    </property>
    <widget class="QLabel" name="decoder_status_label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>116</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>DECODE ERROR</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Decoder:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Symbol stream:</string>
     </property>
    </widget>
    <widget class="QLabel" name="symbol_status_label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>116</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>NO FLOW</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Frame check:</string>
     </property>
    </widget>
    <widget class="QLabel" name="sync_status_label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>116</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>BAD FRAME</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="format_status_label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>116</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>BAD FORMAT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>96</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Format check:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>146</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Frames:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>146</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Messages:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="frame_counter_label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>116</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>--</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="message_counter_label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>116</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>--</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="version_label">
    <property name="geometry">
     <rect>
      <x>725</x>
      <y>695</y>
      <width>316</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ver</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="host_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>700</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>--</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1066</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_TM_to_csv"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSave_TM_to_csv">
   <property name="text">
    <string>Save TM to .csv</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
